### ISSUE TITLE
[Bug]: Layout issues when running under wayland

### ISSUE THREAD

**Issue by wmlele** [mattermost/desktop#3078]
### Checks before filing an issue

- [X] This issue doesn't reproduce on web browsers (such as in Chrome). If it does, [issue reports go to the Mattermost Server repository](https://github.com/mattermost/mattermost-server/issues).
- [X] I have checked the [issue tracker](https://github.com/mattermost/desktop/issues) and have not found an issue that matches the one I'm filing.
- [X] This issue is not a troubleshooting question. Troubleshooting questions go here: https://forum.mattermost.com/c/trouble-shoot/16.
- [X] This issue is not a feature request. You can request features and make product suggestions here: https://mattermost.com/suggestions/.
- [X] This issue reproduces on the most recent [stable version](https://github.com/mattermost/desktop/releases/latest), or the most recent [prerelease version](https://github.com/mattermost/desktop/releases) of the Mattermost Desktop App.
- [X] I have read the [contribution guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md).

### Mattermost Desktop Version

5.7.0

### Operating System

openSUSE Leap 15.6

### Mattermost Server Version

9.8.1

### Steps to reproduce

Running mattermost-desktop on Linux under wayland with 
```
ELECTRON_OZONE_PLATFORM_HINT=wayland
```

Fractional scaling might also be a factor.

### Expected behavior

Layout should remain the same regardless of the rendering platform.

### Observed behavior

The layout changes as soon as the application loses focus.

Layout of the input bar at startup (or after a manual reload)
![Screenshot from 2024-06-26 09-40-52](https://github.com/mattermost/desktop/assets/1617799/dfab5e02-726b-42cd-bd45-64b726cbd7f0)

As soon as I click on another app and the mattermost-desktop is unfocused
![Screenshot from 2024-06-26 09-38-46](https://github.com/mattermost/desktop/assets/1617799/093362cc-ab4f-43b3-8dcf-04f12f5767bd)

### Log Output

```shell
[no relevant log output]
```

**Comment by C-Peiter** [comment]
I experience the same issue using Ubuntu 22.04, Wayland and fractional scaling. It only appears when I enable `WaylandWindowDecorations` (i.e., with the standard Gnome Titlebar). Disabling it fixes the issue, but requires workarounds to move the window using the mouse.

**Comment by devinbinnie** [comment]
So I can reproduce this when setting scaling <100%, however this doesn't appear to be unique to our app, as I observed the same behaviour happening in VSCode, which is also an Electron application.

I would say try to reproduce the issue using an Electron fiddle and report it there, as we aren't changing the size of the window on focus, only the inner views.

Closing as there's nothing we can do from our end.
